website/docs/r/pki_secret_backend_cert.html.md

```diff
@@ -67,7 +67,9 @@ The following arguments are supported:
 
 * `auto_renew` - (Optional) If set to `true`, certs will be renewed if the expiration is within `min_seconds_remaining`. Default `false`
  
-* `revoke` - If set to `true`, the certificate will be revoked on resource destruction. 
+* `revoke` - If set to `true`, the certificate will be revoked on resource destruction. Needs privileged access on Vault engine. Conflicts with `revoke_with_key`. Default `false`.
```

Comment:
> Needs privileged access on Vault engine.

Sorry I meant to call this out in my initial review but I seem to have missed it. This might lead to people thinking that `revoke_with_key` doesn't need to be authorized in Vault which is does and is up to an operator's policy decisions on making it more privileged or not. 

Could we update this to

 ```suggestion
* `revoke` - If set to `true`, the certificate will be revoked on resource destruction using the `revoke` PKI API. Conflicts with `revoke_with_key`. Default `false`.

* `revoke_with_key` - If set to `true`, the certificate will be revoked on resource destruction using the `revoke-with-key` PKI API. Conflicts with `revoke`. Default `false`
```